### DISCUSSION TITLE
While navigating the maps jumps every second to follow the path

### DISCUSSION THREAD

**Discussion by jbslux** [osmandapp/OsmAnd#17812] [Maps Related]
Is there a way to make the map scroll fluently like in google maps while navigating?

i have tried then "animate" option but then it's a mixture of smooth scrolling and jumps.

thanks

**Comment by Sonwon1** [comment]
A lot depends on your hardware. Turn off all POI's, contour lines, etc. and see if that helps. If that works then you could enable them one feature at a time to find which one lags the hardware. Worst case is you will need better hardware.

Unless there is a bug?

**Reply by jbslux** [reply]
Thanks for your reply.
So just to be sure, if you navigate with Osmand Maps, you see the map update and scroll as fluently as in Googlemaps? 

I don't think it's a bug, or a hardware issue (It's happening on my iPhone 14 Pro and Carplay), I think it's the way it's programmed? When I drive and look at the map, then the map scrolls intermittently...

**Reply by Sonwon1** [reply]
Never happened to me on Android.

**Comment by vshcherb** [comment]
There is an open issue for navigation with 3D relief on. Without 3D relief it should be smooth in 4.5.9.

**Comment by jbslux** [comment]
I've made video. You can clearly see what I mean. Thanks for the help.
https://youtu.be/avnpXHmfE2E

**Comment by Sonwon1** [comment]
It looks like a hardware issue although it could be software too. Check the
help info section in the app.


At any moment, you have a choice, that leads you closer to God or further
from God.

**Comment by vshcherb** [comment]
The issue has been fixed in nightly, waiting for 4.5.10 beta / release
